File: T8. DATA TYPES AND VARIABLES - EXERCISES (Java)/src/P13_VowelOrDigit.java
import java.util.Scanner;

public class P13_VowelOrDigit {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        char input = in.nextLine().charAt(0);

        if (input == 'a' || input == 'e' || input == 'o' || input == 'u' || input == 'i') {
            System.out.println("vowel");
        } else if (Character.isDigit(input)) {
            System.out.println("digit");
        } else {
            System.out.println("other");
        }
    }
}

File: T8. DATA TYPES AND VARIABLES - EXERCISES/P13.VowelOrDigit.cs
using System;

namespace P13.VowelOrDigit
{
    class Program
    {
        static void Main(string[] args)
        {
            char input = char.Parse(Console.ReadLine());

            if (input == 'a' || input == 'e' || input == 'o' || input == 'u' || input == 'i')
            {
                Console.WriteLine("vowel");
            }
            else if (char.IsDigit(input))
            {
                Console.WriteLine("digit");
            }
            else
            {
                Console.WriteLine("other");
            }
        }
    }
}
